### ISSUE TITLE
query on url for fetching google site page

### ISSUE THREAD

**Issue by DanishBaloch010** [Arthurpinange/pygoogle_image#4]
hey, 

 url = 'https://www.google.com/search?q=' + quote(
            keyword_to_search[i].encode(
                'utf-8')) + '&biw=1536&bih=674&tbm=isch&sxsrf=ACYBGNSXXpS6YmAKUiLKKBs6xWb4uUY5gA:1581168823770&source=lnms&sa=X&ved=0ahUKEwioj8jwiMLnAhW9AhAIHbXTBMMQ_AUI3QUoAQ'
        raw_html = _download_page(url)

can you please explain how you came up with this link? is there anything modifiable? like security tokens etc ?

why you did not used a simple google image search query like "https://www.google.com/search?q=cars" ... ?

**Comment by populated** [comment]
> hey,
> 
> url = 'https://www.google.com/search?q=' + quote( keyword_to_search[i].encode( 'utf-8')) + '&biw=1536&bih=674&tbm=isch&sxsrf=ACYBGNSXXpS6YmAKUiLKKBs6xWb4uUY5gA:1581168823770&source=lnms&sa=X&ved=0ahUKEwioj8jwiMLnAhW9AhAIHbXTBMMQ_AUI3QUoAQ' raw_html = _download_page(url)
> 
> can you please explain how you came up with this link? is there anything modifiable? like security tokens etc ?
> 
> why you did not used a simple google image search query like "https://www.google.com/search?q=cars" ... ?

No, those other query parameters are useless. The only needed parameters are `q`, and `tbm`.
